12-solid/4-interface-segregation-principle/1-before.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+interface WorkerInterface
+{
+    public function work();
+
+    public function sleep();
+}
+
+class HumanWorker implements WorkerInterface
+{
+    public function work()
+    {
+        return 'Human working.';
+    }
+
+    public function sleep()
+    {
+        return 'Human sleeping.';
+    }
+}
+
+class AndroidWorker implements WorkerInterface
```

Comment:
Whats an Android Worker between? RobotWorker?

Reply:
yeah, if RobotWorker makes more scene let's go with it